catkin_ws/src/state_estimation/launch/state_estimation.launch

```diff
@@ -1,6 +1,6 @@
 <launch>
 	<include file="$(find imu)/launch/imu.launch"/>
-    <include file="$(find depth_sensor)/launch/depth_sensor.launch"/>
+    <!-- <include file="$(find depth_sensor)/launch/depth_sensor.launch"/> -->
```

Comment:
We shouldn't have this commented out on noetic

Reply:
yep